app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomAdapter.kt

```diff
@@ -152,9 +164,25 @@ class ChatRoomAdapter(private val roomType: String,
                                     .show()
                         }
                     }
-                    is AttachmentType.Video,
-                    is AttachmentType.Audio -> {
+                    AttachmentType.Video,
+                    AttachmentType.Audio -> {
                         videoVisible = true
+                        if (message.attachmentType == AttachmentType.Video) {
+                            Observable.fromCallable {
```

Comment:
Please create a `fun` with a meaningful name returning the Observable.

Also, you need to unsubscribe on view recycling, or you can end up having the wrong preview when scrolling the list...

(This doesn't happens right now because we have a bug on the adapter, that will be fixed on a PR I'm sending today)

Reply:
Sure! I was unable to work as there was some problem with my laptop.